src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
+import { checkIncludes } from './helpers/helpers';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => (
+    checkIncludes(movie.description, query)
+    || checkIncludes(movie.title, query)
+  ));
```

Comment:
You can destructure the `movie` variable to improve readability:slightly_smiling_face: